src/TimeSpans.jl

```diff
@@ -119,34 +129,39 @@ function shortest_timespan_containing(spans)
 end
 
 """
-    TimeSpans.duration(span)
+    duration(span)
 
 Return `stop(span) - start(span)`.
 """
 duration(span) = stop(span) - start(span)
 
+"""
+    nanoseconds_per_sample(sample_rate)
+
+Given `sample_rate` in Hz, return the number of nanoseconds corresponding to one sample.
+"""
 nanoseconds_per_sample(sample_rate) = inv(sample_rate) * 1_000_000_000
```

Comment:
This isn't intended to be an exposed API function. I haven't thought about it much but my intuition says that callers should be using `index_from_time`/`time_from_index` and not this function directly